Makefile

```diff
@@ -5,20 +5,28 @@ libtransistor_OBJECT_NAMES := crt0_common.o svc.o ipc.o tls.o util.o ipc/sm.o ip
 libtransistor_OBJECT_FILES := $(addprefix $(LIBTRANSISTOR_HOME)/build/lib/,$(libtransistor_OBJECT_NAMES))
 
 # for building newlib
-export LD = ld.lld$(LLVM_POSTFIX)
-export CC = clang$(LLVM_POSTFIX)
-export CXX = clang++$(LLVM_POSTFIX)
-export AS = llvm-mc$(LLVM_POSTFIX)
-export AR_FOR_TARGET = llvm-ar$(LLVM_POSTFIX)
-export AS_FOR_TARGET = llvm-mc$(LLVM_POSTFIX) -arch=aarch64 -mattr=+neon
-export LD_FOR_TARGET = ld.lld$(LLVM_POSTFIX)
+export LD
```

Comment:
These defines shouldn't be explicitly set and then exported here; they're already set in `libtransistor.mk`. Setting and exporting here will cause an issue if the value is updated in `libtransistor.mk` and not here.